doc/api/crypto.md

```diff
@@ -5457,6 +5460,12 @@ If at least one of `a` and `b` is a `TypedArray` with more than one byte per
 entry, such as `Uint16Array`, the result will be computed using the platform
 byte order.
 
+<strong class="critical">When both of the inputs are `Float32Array`s or
```

Comment:
This section is still incorrect and misleading. If the float/double values are the same, then `timingSafeEqual()` will return `true`. If the values differ, then `timingSafeEqual()` will return `false`.

`0` and `-0` are different values. The fact that the `===` operator treats them as being equal is a language "issue", much like how `==` can return `true` for two different value types due to type coercion.

`timingSafeEqual()` is doing the correct and expected comparison here.

Reply:
Which part is incorrect? The test that's added in this PR demonstrates that `timingSafeEqual()` can return false even when `Object.is()` returns true.

Reply:
ping @mscdex